platform/agc-platform-core/src/main/resources/bootstrap-agents.json

```diff
@@ -79,5 +79,15 @@
       }
     ],
     "suggestedQueries": ["What are the key realisations on AI adoption mentioned in the report."]
+  },
+  {"name":"AutonomousRevenueRetentionAgent",
+    "description":"Demonstrate how AgC can agentify an existing Telco system to autonomously detect and mitigate revenue leakage and churn risk.",
+    "systemPrompt":"System prompt: ROLE\nYou are Customer-Retention Assistant, a concise decision-maker that orchestrates two tools to fulfil retention-related requests.\n\nAvailable tools:\n1. select_retention_cohort: Return a redacted list of customers who meet retention-risk criteria.\n2. apply_retention_actions: Execute one retention action for a batch of customers identified only by customer_id_hash\n\nCore rules\n1. Validate inputs \n    – do not call a tool until every required parameter is present and well-formed or executed with defult values while inform to user\n2. Enforce order \n    – always call select_retention_cohort first; call apply_retention_actions only after a valid cohort is returned.\n3. Single action \n    – each apply_retention_actions call must specify exactly one action value and the customer list from the previous step.\n4. Crisp answers \n    – respond factually and concisely; for purely informational questions, answer directly without invoking tools.\n5. Error handling \n    – if parameters are missing, reply with a clear statement of what is required.",
+    "tools":[
+      {"type":"function","description":"Returns a redacted list of customers who meet specified retention-risk criteria (revenue threshold, recent payment failures, unresolved complaints","name":"select_retention_cohort","parameters":{"type":"object","properties":{"min_revenue":{"type":"number","default":100000},"region":{"type":"string","default":"IN"},"lookback_days":{"type":"number","default":2},"min_payment_failures":{"type":"number","default":3},"min_open_complaints":{"type":"number","default":2}},"required":["min_revenue","region","lookback_days","min_payment_failures","min_open_complaints"],"additionalProperties":false},"strict":true},
```

Comment:
Problems:
1. The loaded functions: select_retention_cohort and apply_retention_actions lacks.... execution_specs {} object.
2. Still on AgC UI they are recognised as client side functions... fix this as well.